### PR TITLE
Add total_dos to phonopy flow

### PR DESCRIPTION
**Pull request by tomdemeyere**
## Summary of Changes

Some help for https://github.com/Quantum-Accelerators/quacc/pull/1655 !

### Checklist

- [ ] I have read the ["Guidelines" section](https://quantum-accelerators.github.io/quacc/dev/contributing.html#guidelines) of the contributing guide. Don't lie! 😉
- [ ] My PR is on a custom branch and is _not_ named `main`.
- [ ] I have used `black`, `isort`, and `ruff` as described in the [style guide](https://quantum-accelerators.github.io/quacc/dev/contributing.html#style).
- [ ] I have added relevant, comprehensive [unit tests](https://quantum-accelerators.github.io/quacc/dev/contributing.html#unit-tests).

### Notes

- Your PR will likely not be merged without proper and thorough tests.
- If you are an external contributor, you will see a comment from [@buildbot-princeton](https://github.com/buildbot-princeton). This is solely for the maintainers.
- When your code is ready for review, ping one of the [active maintainers](https://quantum-accelerators.github.io/quacc/about/contributors.html#active-maintainers).

### CODE REVIEW

**Comment by buildbot-princeton** [comment]
Can one of the admins verify this patch?

File: src/quacc/atoms/phonons.py [hunk 1]
```diff
@@ -62,7 +62,7 @@ def get_phonopy(
         n_supercells = np.round(np.ceil(min_length / atoms.cell.lengths()))
         supercell_matrix = np.diag([n_supercells, n_supercells, n_supercells])
     else:
-        supercell_matrix = None
```
Comment (Andrew-S-Rosen) [inline src/quacc/atoms/phonons.py:65]: This logic should all be taken care of in #1654.

**Comment by Andrew-S-Rosen** [comment]
Thanks, I'll manually incorporate the suggestions.

File: src/quacc/atoms/phonons.py (hunk 1 above)
Comment (tomdemeyere) [reply src/quacc/atoms/phonons.py:65]: Yes sorry I forgot that this was here, changing back

**Pull request merged**